.size-snapshot.json

```diff
@@ -1,39 +1,39 @@
 {
   "build/cytoscape.umd.js": {
```

Comment:
Could you reset this file?  It's only for releases.  Given the confusion it creates, it might be even better to not generate this file at all any more.